main.py

```diff
@@ -12,8 +12,19 @@
 from fastapi.openapi.docs import get_swagger_ui_html, get_swagger_ui_oauth2_redirect_html
 from fastapi.responses import HTMLResponse
 from pydantic import BaseModel
+from typing import Optional
 
 
+class CompilerErrorModel(BaseModel):
+    status: str = "failed"
+    message: str
+    column: Optional[int] = None
+    line: Optional[int] = None
+    error_type: str
```

Comment:
import this from `pydantic`
```suggestion
    error_type: str = Field(alias="errorType")
```